src/dev/impl/DevToys/Views/Tools/Text/XmlValidator/XmlValidatorToolPage.xaml

```diff
@@ -0,0 +1,95 @@
+﻿<Page
+    x:Class="DevToys.Views.Tools.XmlValidator.XmlValidatorToolPage"
+    xmlns="http://schemas.microsoft.com/winfx/2006/xaml/presentation"
+    xmlns:d="http://schemas.microsoft.com/expression/blend/2008"
+    xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml"
+    xmlns:mc="http://schemas.openxmlformats.org/markup-compatibility/2006"
+    xmlns:controls="using:DevToys.UI.Controls"
+    xmlns:ex="using:DevToys.UI.Extensions"
+    xmlns:toolkit="using:Microsoft.Toolkit.Uwp.UI.Controls"
+    xmlns:muxc="using:Microsoft.UI.Xaml.Controls"
+    xmlns:converters="using:DevToys.UI.Converters"
+    mc:Ignorable="d"
+    NavigationCacheMode="Required">
+    
+    <Page.Resources>
+        <converters:NullToBooleanConverter x:Name="NullToVisible" IsInverted="True"/>
+    </Page.Resources>
+
+    <Grid Margin="0,0,0,16">
+        <VisualStateManager.VisualStateGroups>
+            <VisualStateGroup>
+                <VisualState>
+                    <VisualState.StateTriggers>
+                        <ex:IsCompactOverlayModeTrigger TargetElement="{x:Bind}" />
```

Comment:
In Compact Overlay mode, the code editor isn't wide enough to read the XML/XSD easily.

![image](https://user-images.githubusercontent.com/3747805/173728640-51879a4e-c9c7-4b00-a44a-76d710419023.png)

I'd suggest to automatically switch the UI to a horizontal layout when in Compact Overlay mode, like the XML Formatter does:

![image](https://user-images.githubusercontent.com/3747805/173728754-4f3c4cb4-9499-4a00-bcb0-22b06f3ba367.png)

![image](https://user-images.githubusercontent.com/3747805/173728841-d119b8c7-2d13-4c0a-88a1-38d320fd2e58.png)

Here is how it's made:
https://github.com/veler/DevToys/blob/f55c5fb7d56e663cd92561fea219eee6d0ddf053/src/dev/impl/DevToys/Views/Tools/Formatters/XmlFormatter/XmlFormatterToolPage.xaml#L22-L31

Reply:
You are right - was not really convenient that way.
Adjusted the view to look like your suggestion now 🎆

Reply:
Awesome, thanks!